src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/parser/format/MetricCompactJsonFormat.kt

```diff
@@ -0,0 +1,43 @@
+package com.atlassian.performance.tools.jiraactions.api.parser.format
+
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.ActionResult
+import java.time.Duration
+import java.time.Instant
+import java.util.*
+import javax.json.Json
+import javax.json.JsonObject
+
+/**
+ * Format ignoring drilldown parameter
+ */
+class MetricCompactJsonFormat: MetricJsonFormat {
```

Comment:
Can we get add a test case to `ActionMetricsParserTest` just to see if this doesn't throw?